jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -42,7 +42,7 @@
 /**
  * Assert that specified object type value is valid
  *
- * @param object's implementation-defined type
+ * @param type object's implementation-defined type
```

Comment:
Because `@param` needs the name of the parameter it is documenting (which is type in this case).